internal/pkg/config/v1/feature_flags.go

```diff
@@ -17,3 +17,24 @@ type FeatureFlags struct {
 	LastUpdateTime int64                  `json:"last_update_time" hcl:"last_update_time"`
 	User           lduser.User            `json:"user" hcl:"user"`
 }
+
+// ResolveToLaunchDarklyClient resolves to a LaunchDarkly client based on the string platform name that is passed in. It
+// defaults to CliLaunchDarklyClient (CLI LaunchDarkly project).
+func ResolveToLaunchDarklyClient(platformName string) LaunchDarklyClient {
+	switch platformName {
+	case "confluent.cloud":
+		return CcloudProdLaunchDarklyClient
+	case "stag.cpdev.cloud":
+		return CcloudStagLaunchDarklyClient
+	case "devel.cpdev.cloud":
+		return CcloudDevelLaunchDarklyClient
+	default:
+		return CliLaunchDarklyClient
+	}
+}
```

Comment:
Let's only use this for the CC LD client!
```suggestion
func GetCcloudLaunchDarklyClient(platform string) LaunchDarklyClient {
	switch platform {
	case "stag.cpdev.cloud":
		return CcloudStagLaunchDarklyClient
	case "devel.cpdev.cloud":
		return CcloudDevelLaunchDarklyClient
	default:
		return CcloudProdLaunchDarklyClient
	}
}
```

Reply:
Umm if it's a on-prem login, then wouldn't it default to `CcloudProdLaunchDarklyClient` and `admin payment describe` would theoretically try to get ccloud prod's flag value?

Though `admin payment describe` wouldn't even be able to run because the pre-runner requires cloud login...

Reply:
Yep, the command will only run if you're logged in to Cloud!

Reply:
sure thing. lmk if the latest commit looks good to you!